### PR TITLE
feat(components-native): Import InputCurrency

### PR DESCRIPTION
**Pull request by MichaelParadis**
### Added

- feat(components-native): Added InputCurrency
- feat(components-native): Added currencySymbol to AtlantisContext

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- run `npm run pack @jobber/components-native`
- Install into Mobile project
- Replace InputCurrency with the new InputCurrency and verify it works

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components-native/src/AtlantisContext/AtlantisContext.test.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ const providerValues: AtlantisContextProps = {
     return;
   },
   floatSeparators: { decimal: ".", group: "," },
+  currencySymbol: "€",
```
Comment (rodrigoeidelvein) [inline packages/components-native/src/AtlantisContext/AtlantisContext.test.tsx:22]: Any specific reason to use this symbol?

Comment (MichaelParadis) [reply packages/components-native/src/AtlantisContext/AtlantisContext.test.tsx:22]: To test that it is actually using provided values and not just using the default

File: packages/components-native/src/AtlantisContext/AtlantisContext.tsx [hunk 2]
```diff
@@ -2,6 +2,8 @@
 import { createContext, useContext } from "react";
 import RNLocalize from "react-native-localize";
 
+export const DEFAULT_CURRENCY_SYMBOL = "$";
```
Comment (JeffMarvin) [inline packages/components-native/src/AtlantisContext/AtlantisContext.tsx:5]: I wonder if this constant should sit somewhere more relevant... maybe in InputCurrency (the component or a constants file in that folder)?

**Comment by JeffMarvin** [comment]
@MichaelParadis Code looks good apart from a ponderous question about where to define the currency symbol constant. 
I will run it through a quick QA pass.

**Review by JeffMarvin — APPROVED** [review]
Tested on mobile with the Payment Overlay and things are looking good!

**Pull request merged**